pom.xml

```diff
@@ -73,4 +74,41 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>checkstyle</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-checkstyle-plugin</artifactId>
+                        <version>${maven.checkstyle.plugin.version}</version>
+                        <executions>
+                            <execution>
+                                <phase>validate</phase>
+                                <goals>
+                                    <goal>checkstyle</goal>
```

Comment:
Can you update readme how to locally validate checkstype? `mvn clean validate` I guess before the compilation. That is mostly asked by community when they hit some checkstyle issues.